tests/tests.rs

```diff
@@ -3,33 +3,39 @@ mod tester;
 use tester::BatTester;
 
 macro_rules! snapshot_tests {
-    ($($test_name: ident: $style: expr,)*) => {
+    ($($test_name: ident: $style: expr => [wrap: $wrap:expr, tabs: $tabs:expr],)*) => {
         $(
             #[test]
             fn $test_name() {
                 let bat_tester = BatTester::new();
-                bat_tester.test_snapshot($style);
+                bat_tester.test_snapshot(stringify!($test_name), $style, $tabs, $wrap);
             }
         )*
     };
 }
 
 snapshot_tests! {
-    changes: "changes",
-    grid: "grid",
-    header: "header",
-    numbers: "numbers",
-    changes_grid: "changes,grid",
-    changes_header: "changes,header",
-    changes_numbers: "changes,numbers",
-    grid_header: "grid,header",
-    grid_numbers: "grid,numbers",
-    header_numbers: "header,numbers",
-    changes_grid_header: "changes,grid,header",
-    changes_grid_numbers: "changes,grid,numbers",
-    changes_header_numbers: "changes,header,numbers",
-    grid_header_numbers: "grid,header,numbers",
-    changes_grid_header_numbers: "changes,grid,header,numbers",
-    full: "full",
-    plain: "plain",
+    changes:                     "changes"                     => [wrap: false, tabs: 8],
+    grid:                        "grid"                        => [wrap: false, tabs: 8],
+    header:                      "header"                      => [wrap: false, tabs: 8],
+    numbers:                     "numbers"                     => [wrap: false, tabs: 8],
+    changes_grid:                "changes,grid"                => [wrap: false, tabs: 8],
+    changes_header:              "changes,header"              => [wrap: false, tabs: 8],
+    changes_numbers:             "changes,numbers"             => [wrap: false, tabs: 8],
+    grid_header:                 "grid,header"                 => [wrap: false, tabs: 8],
+    grid_numbers:                "grid,numbers"                => [wrap: false, tabs: 8],
+    header_numbers:              "header,numbers"              => [wrap: false, tabs: 8],
+    changes_grid_header:         "changes,grid,header"         => [wrap: false, tabs: 8],
+    changes_grid_numbers:        "changes,grid,numbers"        => [wrap: false, tabs: 8],
+    changes_header_numbers:      "changes,header,numbers"      => [wrap: false, tabs: 8],
+    grid_header_numbers:         "grid,header,numbers"         => [wrap: false, tabs: 8],
+    changes_grid_header_numbers: "changes,grid,header,numbers" => [wrap: false, tabs: 8],
+    full:                        "full"                        => [wrap: false, tabs: 8],
+    plain:                       "plain"                       => [wrap: false, tabs: 0],
```

Comment:
@eth-p I had to change this to `tabs: 0` because `generate_snapshots.py` produced a file that had tabs in them for `--style=plain`. I'm not sure why this was different for you. I tried to add `--paging=never` and `--color=never` to make this more robust.

Reply:
@sharkdp That would be because it doesn't process tabs unless it needs to for alignment reasons (e.g. the pager or decorations are used). On second thought, though, that might be a bit confusing when people inconsistently end up with tabs sometimes.

Should I make a PR to remove that?

Reply:
I think the current behavior is fine. I just wondered why I got a different version of the file when I ran the `generate_..` script.